api/test/unit/serializers/media_serializers_test.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+from unittest.mock import MagicMock
+
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+
+import pytest
+
+from catalog.api.serializers.audio_serializers import AudioSerializer
+from catalog.api.serializers.image_serializers import ImageSerializer
+
+
+@pytest.fixture
+def req():
+    factory = APIRequestFactory()
+    request = factory.get("/")
+    request = Request(request)
+    return request
+
+
+@pytest.fixture
+def hit():
+    hit = MagicMock(
+        identifier=uuid.uuid4(),
+        license="cc0",
+        license_version="1.0",
+    )
+    return hit
+
+
+@pytest.mark.parametrize(
+    "serializer_class",
+    [
+        AudioSerializer,
+        ImageSerializer,
+    ],
```

Comment:
FWIW it's possible to use `__subclasses__` for this so it automatically updates itself when we add new serializers in the future.

Reply:
Subclasses are only registered when the serializers are imported. So as I see it, you'll still need to import the serializer files for the side-effect of registering subclasses.

<img width="521" alt="Screenshot 2022-08-19 at 8 50 59 AM" src="https://user-images.githubusercontent.com/16580576/185535836-3b135ae9-d6b4-411e-b9b5-5bb489c3d228.png">

Reply:
Ahhh, they're not in the same modules. Rats!